docs/scos/dev/guidelines/keeping-a-project-upgradable/keeping-a-project-upgradable.md

```diff
@@ -1,18 +1,19 @@
 ---
 title: Keeping a project upgradable
-description: Tools and guidelines for keeping a project upgradable
+description: Guidelines for keeping a project upgradable
+last_updated: Mar 24, 2023
 template: concept-topic-template
 related:
-  - title: Upgradability guidelines
-    link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/upgradability-guidelines.html
   - title: Upgrader tool overview
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgrader-tool-overview.html
-  - title: Run the evaluator tool
-    link: docs/scos/dev/guidelines/keeping-a-project-upgradable/run-the-evaluator-tool.html
-  - title: Running the upgrader tool
-    link: docs/scos/dev/guidelines/keeping-a-project-upgradable/run-the-upgrader-tool.html
-  - title: Define custom prefixes for core entity names
-    link: docs/scos/dev/guidelines/keeping-a-project-upgradable/define-customs-prefixes-for-core-entity-names.html
+  - title: Run the Upgrader tool
+    link:/docs/scos/dev/guidelines/keeping-a-project-upgradable/run-the-upgrader-tool.html
```

Comment:
Please remove / from the link before "docs". This might cause the build to fail and tell you there is an issue with the template